docs/usage.md

```diff
@@ -47,7 +47,7 @@ The *Feed Ingested* dashboard component reflects statistics for Threat Intel Fee
 
 Using a wide variety of feed integrations that are available on the **Content Hub**, you can seamlessly ingest feeds and get a normalized and aggregated view of the feeds on the **Threat Feeds** tab:  
 
-![Threat Feeds tab](res/threat-feeds.png)
+![Threat Feeds tab](res/threat-feeds.png) <!-- Change Screenshot -->
```

Comment:
where are we explaining new landing page? navigation updates? first class dashboard in dashboard drop down?, etc.